analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/PublicMethodArgumentsShouldBeCheckedForNull.CSharp8.cs

```diff
@@ -97,7 +97,7 @@ public void MultipleBranches_HandleNull(string s, int val)
     {
         var result = s switch
         {
-            null => s.ToString(),   // FIXME Non-compliant
+            null => s.ToString(),   // FN
```

Comment:
Will this not be fixed?
Same question for other tests where you changed `FIXME Non-compliant` to `FN`.

Reply:
Most likely not. Definitely not in this PR, as it's only about the most basic implementation of the rule.
I tried these FN cases in the debugger, but the parameter's null state was always set to `Null` or `NotNull` by the engine, so I'm not sure how to handle them in the rule.

Reply:
I looked at it again: This should be compliant because it is covered by S2259.